wagtail/admin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -135,11 +135,16 @@ li.sequence-member {
 // that are direct children of highest level sequence should be borderless and full-width
 .block_field > .field-content > .input > .sequence-container > .sequence-container-inner > .sequence > .sequence-member > .sequence-member-inner {
     > .widget-text_input input,
-    > .widget-hallo_rich_text_area .richtext,
+    > .halloeditor,
     > .widget-textarea textarea {
-        border: 0;
         padding: 0;
+        max-width: 1024px;
+        border: 0;
         background-color: transparent;
+    }
+
+    > .Draftail-Editor {
```

Comment:
Shouldn’t we stick to lowercase?

Reply:
No, the React best practice of the day is to use the component's name as the class name, with the same case if possible. For some reason that escapes me, React components are TitleCased. For `Draftail-`, it's a namespace for all Draftail CSS.

Reply:
OK, thanks for explaining @thibaudcolas :)